src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -605,6 +605,15 @@
 	IEmoji getEmojiByName(String name);
 
 	/**
+	 * This adds a custom emoji to the guild.
+	 *
+	 * @param name The name, <b>without colons</b>.
+	 * @param image The BufferedImage of the Emoji.
+	 * @return The emoji.
```

Comment:
[If it can throw MissingPermissionsException, DiscordException and RateLimitException then it should be listed in the javadocs that it does. You can see in this search that it is done everywhere else](https://github.com/austinv11/Discord4J/search?utf8=%E2%9C%93&q=%40throws+DiscordException+RateLimitException+MissingPermissionsException&type=)